src/EditorFeatures/Core/Extensibility/Commands/PredefinedCommandHandlerNames.cs

```diff
@@ -88,6 +88,11 @@ internal static class PredefinedCommandHandlerNames
         public const string GoToDefinition = "Go To Definition Command Handler";
 
         /// <summary>
+        /// Command handler name for Go to Implementation.
+        /// </summary>
+        public const string GoToImplementation = "Go To Definition Command Handler";
```

Comment:
Oops. I _may_ have also forgotten to consume this string versus the GoToDefinition one. Just maybe.